docs/source/top_level_elements/location.rst

```diff
@@ -0,0 +1,208 @@
+        <holdingSimple>
+            <copyInformation>
+                <shelfLocator>Manuscript Number 5</shelfLocator>
+            </copyInformation>
+        </holdingSimple>
+        <holdingExternal>
+            <holding xsi:schemaLocation="info:ofi/fmt:xml:xsd:iso20775 http://www.loc.gov/standards/iso20775/N130_ISOholdings_v6_1.xsd">
+                <physicalAddress>
+                    <text>City: Memphis</text>
+                    <text>County: Shelby County</text>
+                    <text>State: Tennessee</text>
+                </physicalAddress>
+            </holding>
+        </holdingExternal>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+    @prefix opaque: <http://opaquenamespace.org/ns/> .
+
+    <https://example.org/objects/1> relators:rps "University of Memphis. Special Collections" ;
+        opaque:locationShelfLocator "Manuscript Number 5" .
+
+physicalLocation with displayLabel="Address"
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+https://digital.lib.utk.edu/collections/islandora/object/arrow:58/datastream/MODS/view
+
+.. code-block:: xml
+
+
+    <location>
+        <physicalLocation>Pi Beta Phi Fraternity</physicalLocation>
+        <physicalLocation displayLabel="Address">1154 Town and Country Commons Drive, Town and Country, Missouri 63017</physicalLocation>
+        <shelfLocator>Box 36, Folder 14</shelfLocator>
+    </location>
+
+*This is one where I'm not sure where to go with.*
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+    @prefix opaque: <http://opaquenamespace.org/ns/> .
+
+    <https://example.org/objects/1> relators:rps "Pi Beta Phi Fraternity" ;
+        opaque:locationShelfLocator "Box 36, Folder 14" .
+
+
+physicalLocation with displayLabel attributes for Collection and Repository
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+https://digital.lib.utk.edu/collections/islandora/object/arrowmont%3A208/datastream/MODS/view
+
+.. code-block:: xml
+
+
+    <location>
+        <physicalLocation displayLabel="Collection">Archives Collection</physicalLocation>
+        <physicalLocation displayLabel="Repository">Arrowmont School of Arts and Crafts</physicalLocation>
+        <physicalLocation displayLabel="Detailed Location"/>
+        <physicalLocation displayLabel="City">Gatlinburg</physicalLocation>
+        <physicalLocation displayLabel="State">Tennessee</physicalLocation>
+    </location>
+
+*I am not sold on whether retaining the Archives Collection string is necessary. I don't think the city and state are necessary if a URI is used instead of string.*
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+    @prefix dbo: <http://dbpedia.org/ontology/> .
+
+    <https://example.org/objects/1> relators:rps <http://id.loc.gov/authorities/names/no2001080757> ;
+        dbo:collection "Archives Collection" .
```

Comment:
Are you suggesting that <https://example.org/objects/1> is an "Archives Collection?"

Reply:
@markpbaggett I was suggesting that is in an Archives Collection at http://id.loc.gov/authorities/names/no2001080757 repository. Should we drop this?

Reply:
Sorry, misunderstood.